c2corg_api/views/document.py

```diff
@@ -110,12 +112,29 @@ def _get(self, clazz, schema, clazz_locale=None, adapt_schema=None,
         id = self.request.validated['id']
         lang = self.request.validated.get('lang')
         editing_view = self.request.GET.get('e', '0') != '0'
+        cook = self.request.validated.get('cook')
+
+        if cook and lang:
```

Comment:
no, because the inside logic is different. 

* `/waypoints/123?lang=en` will returns `en` locale, or no locale. 
* And `/waypoints/123?cook=en` will aways returns a locale and a cooked, even if it's not `en`

I figure that a query parameter must not touch the behavior of another query parameter, and prefer to use another one.